v2/models/address.go

```diff
@@ -5,10 +5,58 @@
 
 package models
 
+import (
+	"encoding/json"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2"
+)
+
 type Address interface {
 	GetBaseAddress() BaseAddress
 }
 
+// FormatAddress convert the interface to the corresponding address type
+func FormatAddress(i interface{}) (address Address, err error) {
```

Comment:
use lower case for the func name
rename to `instantiateAddress`

Reply:
Renamed.